tests/test_load_data.py

```diff
@@ -307,6 +310,31 @@ def test_grid_netcdf(self):
 
         self.assertIsNone(run_smrf(config))
 
+class TestLoadTopo(unittest.TestCase):
+    @classmethod
+    def setUp(self):
+        base = dirname(smrf.__file__)
+        self.test_dir = abspath(join(base, '../', 'tests'))
+        topo_config = {
+            'filename': join(self.test_dir, 'RME/topo/topo.nc'),
+            'northern_hemisphere':True,
+        }
+
+        self.topo = loadTopo.topo(
+            topo_config,
+            calcInput=False,
+            tempDir= join(self.test_dir, 'RME/output')
+        )
+
+    def test_auto_calc_lat_lon(self):
```

Comment:
This appears to just be testing if the topo.nc has a mask, lets add a test for when we don't have the mask or turning off that mask to get the center of the domain.